python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -110,6 +112,22 @@ if(WIN32)
     LIST(REMOVE_ITEM TEST_OPS test_ref_by_trainer_id_op)
 endif()
 
+if(NOT WITH_DISTRIBUTE)
```

Comment:
`if(WITH_DISTRIBUTE)`?

Reply:
if NOT WITH_DISTRIBUTE, these three uts need to be removed